velodyne_driver/src/driver/driver.cc

```diff
@@ -92,7 +95,8 @@ VelodyneDriver::VelodyneDriver(ros::NodeHandle node,
     }
   else
     {
-      input_.reset(new velodyne_driver::InputSocket(private_nh));
+      input_.reset(new velodyne_driver::InputSocket(private_nh,
+                                                    config_.udp_port));
```

Comment:
So far as I can tell, the config_ structure isn't available inside of InputSocket, unless I were to pass it in. I figured it'd be easier to just pass the port number directly, so I wouldn't change the signature of InputSocket's constructor.